gcp_variant_transforms/libs/vcf_field_conflict_resolver.py

```diff
@@ -14,27 +14,38 @@
 
 import vcf
 
+
 class VcfParserConstants(object):
   """Constants for type and number from VCF parser."""
   FLOAT = 'Float'
   INTEGER = 'Integer'
+  STRING = 'String'
+  FLAG = 'Flag'
+  CHARACTER = 'Character'
   NUM = 'num'
   TYPE = 'type'
 
+
 class FieldConflictResolver(object):
-  """A class for resolving all VCF field related mistmatches."""
+  """A class for resolving all VCF field related mismatches."""
   # TODO(nmousavi): explain this class more.
 
-  def __init__(self, split_alternate_allele_info_fields=True):
-    # type: (bool) -> None
+  def __init__(self,
+               split_alternate_allele_info_fields=True,
+               resolve_all=False):
+    # type: (bool, bool) -> None
     """Initialize the class.
 
     Args:
-     split_alternate_allele_info_fields: Whether INFO fields with
-       `Number=A` are stored under the alternate_bases record.
-     """
+      split_alternate_allele_info_fields: Whether INFO fields with `Number=A`
+        are stored under the alternate_bases record.
+      resolve_all: Resolves all incompatible conflicts:
+        any incompatible type conflict is resolved as type = `String`
```

Comment:
This is not true, if the conflict is between Integer and Float, we resolve it to Float. 
Maybe say that if this flag set, we always find a resolution, i.e., as a last resort we convert all type conflicts to String and Number to '.'?

Reply:
Thanks! Here I mean all incompatible conflicts while Integer vs. Float are compatible. I realized it was not clear what are incompatible conflicts. I will change based on your comments :)